lib/src/rules/prefer_foreach.dart

```diff
@@ -8,11 +8,16 @@ import 'package:analyzer/dart/element/element.dart';
 import 'package:linter/src/analyzer.dart';
 import 'package:linter/src/util/dart_type_utilities.dart';
 
-const _desc = r'Use forEach call to only apply a function to all the elements.';
+const _desc = r'Use `.forEach` to only apply a function to all the elements.';
```

Comment:
I'd remove the period (both here and below).

Reply:
This certainly isn't the only instance where I used \`.methodName\` as a standard for referencing methods. Should I change all instances to be \`methodName\` instead?

Reply:
That would be _my_ preference, but I won't block the merge because of it.